From 3144653fe541c6db82aa07c3fa8ae48b0bb0d4ab Mon Sep 17 00:00:00 2001
From: Andy Gospodarek <andy@greyhouse.net>
Date: Tue, 14 Aug 2012 14:59:45 -0400
Subject: [PATCH 1/6] compat: RHEL build script fixup from upstream

There are parentheses where there should be curly braces.  This is a
patch that Philip Perry posted to the list, but it was never
integrated.

Signed-off-by: Philip J Perry <phil@elrepo.org>
Signed-off-by: Andy Gospodarek <andy@greyhouse.net>
Signed-off-by: Luis R. Rodriguez <mcgrof@do-not-panic.com>
---
 scripts/gen-compat-config.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/compat/scripts/gen-compat-config.sh b/compat/scripts/gen-compat-config.sh
index b70ef03..a3ebf08 100755
--- a/compat/scripts/gen-compat-config.sh
+++ b/compat/scripts/gen-compat-config.sh
@@ -48,7 +48,7 @@ done
 # The RHEL checks seem to annotate the existance of RHEL minor versions.
 RHEL_MAJOR=$(grep ^RHEL_MAJOR ${KLIB_BUILD}/Makefile | sed -n 's/.*= *\(.*\)/\1/p')
 if [[ ! -z ${RHEL_MAJOR} ]]; then
-	RHEL_MINOR=$(grep ^RHEL_MINOR $(KLIB_BUILD)/Makefile | sed -n 's/.*= *\(.*\)/\1/p')
+	RHEL_MINOR=$(grep ^RHEL_MINOR ${KLIB_BUILD}/Makefile | sed -n 's/.*= *\(.*\)/\1/p')
 	for i in $(seq 0 ${RHEL_MINOR}); do
 		eval CONFIG_COMPAT_RHEL_${RHEL_MAJOR}_${i}=y
 		echo "export CONFIG_COMPAT_RHEL_${RHEL_MAJOR}_${i}=y"
-- 
1.7.11.1

